tests/test_intern_table.py

```diff
@@ -1,15 +1,22 @@
+import sys
+
 from tests.coveragetest import CoverageTest
 
 from hypothesis import given, example
 import hypothesis.strategies as st
-from coverage.tracer import InternTable
-import sys
+
+from coverage import env
+
+if env.C_TRACER:
+    from coverage.tracer import InternTable
```

Comment:
These tests all run under pytest, right? Perhaps importorskip would be better here?

Reply:
/me googles importorskip

Reply:
I'd rather keep the conditional based on C_TRACER, in case the import fails for other reasons.

Reply:
Fair enough. It probably still makes sense to do a module level skip here rather than at the level of individual tests though. e.g. using https://docs.pytest.org/en/latest/skipping.html#skip-all-test-functions-of-a-class-or-module

Reply:
Yup, good call. I have yet to internalize the ways of pytest.